events/class_tests/meeting_tests.py

```diff
@@ -59,3 +60,25 @@ def test_persist_possible_meeting_in_db(self):
     def test_duplicate_possible_meeting(self, persist_possible_meeting):
         with pytest.raises(ValidationError, match='meeting hours already exists'):
             persist_possible_meeting.save()
+
+    def test_get_all_event_dates(self, event_title="event3"):
+        event = Event.objects.get(title=event_title)
+        expected_results = [
+            PossibleMeeting.objects.get(id=3),
+            PossibleMeeting.objects.get(id=4),
+            PossibleMeeting.objects.get(id=5),
+        ]
```

Comment:
These lines are the same as lines 75-80.
Maybe you can write them as fixture to avoid code duplication.